editor/edit-post/sidebar/post-schedule/style.scss

```diff
@@ -3,18 +3,8 @@
 	position: relative;
 }
 
-.editor-post-schedule__toggle.button-link {
-	text-decoration: underline;
-	color: $blue-wordpress;
+.wp-core-ui .editor-post-schedule__toggle.button-link {
 	text-align: right;
-
-	&:focus {
-		outline: none;
```

Comment:
Noting that this style does not exist on the base `.button-link`, but also that it's questionable we'd want it.